storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -260,6 +269,19 @@ static struct my_option my_long_options[] = {
      &opt_parallel, &opt_parallel, 0, GET_ULONG, REQUIRED_ARG, 1, 1, ULONG_MAX,
      0, 0, 0},
 
+    {"max-retries", OPT_MAX_RETRIES,
+     "Number of retires of chunk uploads/downloads after a failure (Default "
+     "3).",
+     &opt_max_retries, &opt_max_retries, 0, GET_ULONG, REQUIRED_ARG, 3, 1,
```

Comment:
Not a strong opinion. I somehow feel 3 is too less. Do you think default should be increased? Say 10?

Reply:
I agree, Will increase to 10